regression/test.pl

```diff
@@ -290,6 +300,11 @@ ($$$$)
              independent logs.
   -f         forward the test name to CMD
 
+  --[no]color enable/disable color output; enabled by default unless
+              TESTPL_COLOR_OUTPUT is set, in which case it is enabled by
+              default if its value is nonempty and nonzero, and enabled
+              by default otherwise.
+
```

Comment:
I may just be being particularly slow today, but I find that message a little difficult to parse :-) Maybe a clearer description might be something like:

enabled by default, unless the TESTPL_COLOR_OUTPUT environment variable is set to a nonempty, nonzero value. This option overrides any value of TESTPL_COLOR_OUTPUT

what do you think?

Reply:
I think it might be hard to parse because I mistyped the latter half of the sentence, it was meant to be

> and disabled by default otherwise.

instead of

> and enabled by default otherwise.

Reply:
I've simplified the text, should be easier to parse now